lib/addons/osb-addons.js

```diff
@@ -65,58 +65,84 @@ async function get_state(pg_pool, addon_definition, plan_id, addon_id, app_uuid)
   }
 }
 
-async function provision(pg_pool, type, app_uuid, app_name, space_name, org_name, addon_plan, service_uuid) {
+async function provision(pg_pool, type, app_uuid, app_name, space_name, org_name, addon_plan, addon_id) {
```

Comment:
COBRA-2979 - A possible problem of not issuing the create/delete bindings during OSB creation may have resulted in the missing `REDIS_URL`.  This also standardizes the naming a bit.